docs/source/parsing.md

```diff
@@ -25,30 +25,39 @@ To parse a collection into a forward index, use `parse_collection` command:
 
 For example:
 
+    $ mkdir -p path/to/forward
     $ zcat ClueWeb09B/*/*.warc.gz | \   # pass unzipped stream in WARC format
         parse_collection \
         -j 8 \                          # use up to 8 threads at a time
         -b 10000 \                      # one thread builds up to 10k documents in memory
         -f warc \                       # use WARC
         --stemmer porter2 \             # stem every term using the Porter2 algorithm
         --content-parser html \         # parse HTML content before extracting tokens
-        -o cw09b
+        -o path/to/forward/cw09b
 
-The above command will write the following files:
-- `cw09b`: forward index in binary format,
+In case you get the error `-bash: /bin/zcat: Argument list too long`, you can pass the unzipped stream using:
+
+    $ find ClueWeb09B -name '*.warc.gz' -exec zcat -q {} \;
+
+The parsing will write the following files:
+- `cw09b`: forward index in binary format.
 - `cw09b.terms`: a new-line-delimited list of sorted terms,
-  where term having ID N is on line N
+  where term having ID N is on line N+1.
 - `cw09b.documents`: a new-line-delimited list of document titles (e.g., TREC-IDs),
-  where document having ID N is on line N
+  where document having ID N is on line N+1.
+- `cw09b.urls`: a new-line-delimited list of URLs, where URL having ID N is on
+  line N+1 and where each id corresponds with an id of the `cw09b.documents` file.
 
-### Supported stemmers:
+### Supported stemmers
 - Porter2
 - Krovetz
 
-### Supported formats:
+### Supported formats
 - `plaintext`: every line contains the document's title first, then any number of
-             whitespaces, followed by the content delimited by a new line character,
-- `trextext`: TREC newswire collections
-- `trecweb`: TREC web collections
-- `warc`: Web ARChive format as defined in [the format specification](https://iipc.github.io/warc-specifications/specifications/warc-format/warc-1.0/)
-- `wapo`: TREC Washington Post Corpus
+             whitespaces, followed by the content delimited by a new line character.
+- `trectext`: TREC newswire collections.
+- `trecweb`: TREC web collections.
+- `warc`: Web ARChive format as defined in [the format specification](https://iipc.github.io/warc-specifications/specifications/warc-format/warc-1.0/).
+- `wapo`: TREC Washington Post Corpus.
+
+In case you want to parse a file collection (for example [wiki-large](http://dg3rtljvitrle.cloudfront.net/wiki-large.tar.gz)) use the `corpusfile2trec.py` script to convert it to TREC. Then, execute the `parse_collection` command by passing the resulting file and specifying the `trectext` format.
```

Comment:
What is "a file collection", is that one document per file? Let's clarify.

Reply:
@amallia, @elshize, based on your comments, I have rewritten this part as:
> In case you want to parse a set of files where each one is a document (for example, the collection [wiki-large](http://dg3rtljvitrle.cloudfront.net/wiki-large.tar.gz)), use the `files2trec.py` script to format it to TREC (take into account that each filename corresponds to a document ID). Once the file is generated, parse it with the `parse_collection` command specifying the `trectext` value for the `--format` option.
Let me know if there is ok.

Reply:
Yep, much clearer now.